Gemfile.lock

```diff
@@ -12,22 +12,23 @@ GEM
     algoliasearch (1.27.5)
       httpclient (~> 2.8, >= 2.8.3)
       json (>= 1.5.1)
+    artifactory (3.0.15)
```

Comment:
What is artifactory used for ? Is it a dep from an other pacakge ?

Reply:
Yes, it's a dependency for `fastlane`